tests/test_core.py

```diff
@@ -1790,6 +1790,11 @@ def test_em_asm_parameter_pack(self):
     self.emcc_args += ['-std=c++11']
     self.do_run_in_out_file_test('tests', 'core', 'test_em_asm_parameter_pack')
 
+  def test_em_asm_arguments_side_effects(self):
+    self.do_run_in_out_file_test('tests', 'core', 'test_em_asm_arguments_side_effects', force_c=True)
+    self.emcc_args += ['-std=c++11']
```

Comment:
Should probably get test in the default mode, not C++03?

Reply:
Indeed. In particular if the constraint is that EM_ASM should work without changing the default C++ standard version (which it should), then that's what we should test.

I believe the `-std=c++11` in the previous test is because the test_em_asm_parameter_pack test itself depends on template parameter packing in user code, whereas test_em_asm_arguments_side_effects itself does not.